pkg/nodeconfig/nodeconfig.go

```diff
@@ -134,6 +134,44 @@ func discoverKubeAPIServerEndpoint() (string, error) {
 	return host.Status.APIServerInternalURL, nil
 }
 
+// getWICDCredentials returns the CA cert and access token associated with the WICD service account
+func getWICDCredentials() (*cluster.Authentication, error) {
+	cfg, err := crclientcfg.GetConfig()
+	if err != nil {
+		return nil, errors.Wrap(err, "unable to get config to talk to kubernetes api server")
+	}
+	clientset, err := kubernetes.NewForConfig(cfg)
+	if err != nil {
+		return nil, errors.Wrap(err, "error creating kubernetes clientset")
+	}
+
+	secrets, err := clientset.CoreV1().Secrets(WMCONamespace).List(context.TODO(),
```

Comment:
This should be a method of nodeconfig and use nc.getNamespace

Reply:
This is to used to initialize the nodeconfig cache. Take a look at the [other functions](https://github.com/openshift/windows-machine-config-operator/blob/master/pkg/nodeconfig/nodeconfig.go#L99) we use there.